contents/docs/how-posthog-works/data-model.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Data model
+---
+
+This document provides a high-level overview of the various objects and primitives that make up the PostHog data model. For more information on exactly how data is stored in our database, check out the [Clickhouse overview](/docs/how-posthog-works/clickhouse).
+
+The two most-basic entities in PostHog are the [Event](#event) and [Person](#person) objects, and represent the core of our analytics functionalities.
+
+## Event
+
+An event is the most important object in PostHog, and represents a single action that a user performed at a specific point in time. These events are sent either from one of our [libraries](/docs/integrate) or directly via our [API](/docs/api).
+
+Each event contains the following base fields:
+
+| Property              | Type                   | Description                                                                                                                                                                 |
```

Comment:
After persons on events we now have a lot more columns actually:

(you can ignore _timestamp and _offset if you like - these are the timestamp and offset of the message coming from the kafka table)

```sql
CREATE TABLE posthog.writable_events (
    `uuid` UUID,
    `event` String,
    `properties` String,
    `timestamp` DateTime64(6, 'UTC'),
    `team_id` Int64,
    `distinct_id` String,
    `elements_hash` String,
    `created_at` DateTime64(6, 'UTC'),
    `_timestamp` DateTime,
    `_offset` UInt64,
    `elements_chain` String,
    `person_id` UUID,
    `person_properties` String,
    `group0_properties` String,
    `group1_properties` String,
    `group2_properties` String,
    `group3_properties` String,
    `group4_properties` String,
    `person_created_at` DateTime64(3),
    `group0_created_at` DateTime64(3),
    `group1_created_at` DateTime64(3),
    `group2_created_at` DateTime64(3),
    `group3_created_at` DateTime64(3),
    `group4_created_at` DateTime64(3)
) ENGINE = Distributed(
    'posthog',
    'posthog',
    'sharded_events',
    sipHash64(distinct_id)
)
```